statistics/handle/update.go

```diff
@@ -278,8 +280,8 @@ func (h *Handle) sweepList() {
 }
 
 // DumpStatsDeltaToKV sweeps the whole list and updates the global map, then we dumps every table that held in map to KV.
-// If the `dumpAll` is false, it will only dump that delta info that `Modify Count / Table Count` greater than a ratio.
-func (h *Handle) DumpStatsDeltaToKV(dumpMode bool) error {
+// If the dumpMode is `DumpDelta`, it will only dump that delta info that `Modify Count / Table Count` greater than a ratio.
+func (h *Handle) DumpStatsDeltaToKV(dumpMode dumpMode) error {
```

Comment:
Use `mode` instead of `dumpMode`, since the type is `dumpMode`.

Reply:
renamed.